discovery-infra/utils.py

```diff
@@ -18,17 +19,35 @@
 conn = libvirt.open("qemu:///system")
 
 
-def run_command(command, shell=False):
+def run_command(command, shell=False, callback=None):
```

Comment:
don't understand the usage of callback in sync function. U are not using it in async manner, why it is needed?

Reply:
if u don't want to raise exception just pass flag for it check=False i think and make it default true